sample-operators/mysql-schema/src/main/java/io/javaoperatorsdk/operator/sample/MySQLSchemaReconciler.java

```diff
@@ -54,8 +55,14 @@ public ErrorStatusUpdateControl<MySQLSchema> updateErrorStatus(MySQLSchema schem
   }
 
 
-  private void updateStatusPojo(MySQLSchema mySQLSchema, Schema schema, String secretName,
+  private MySQLSchema createForStatusUpdate(MySQLSchema mySQLSchema, Schema schema,
+      String secretName,
       String userName) {
+    MySQLSchema res = new MySQLSchema();
```

Comment:
To be honest, having to do that for SSA is quite annoying… I'm wondering if there's anything we can do about it…

Reply:
Yeah, not sure, maybe an utility to clone a shallow class? this results certainly in a kind of a boilerplate code .